x-pack/plugins/index_management/public/application/components/mappings_editor/constants/parameters_definition.tsx

```diff
@@ -502,7 +502,7 @@ export const PARAMETERS_DEFINITION: { [key in ParameterName]: ParameterDefinitio
       }
     ),
     fieldConfig: {
-      defaultValue: '',
+      defaultValue: 1,
```

Comment:
Maybe we should check with an ES engineer to see what a good default value might be; `1` seems a little weird to me, since it's used as a multiplier. The documentation uses `10` as an example.

Docs: https://www.elastic.co/guide/en/elasticsearch/reference/current/number.html#scaled-float-params

Reply:
I would leave it as 1, as multiplying by one does not do anything. The default value should be what ES uses behind the scene in case the parameter is not specified, and I think by default ES does not scale float.

Reply:
Seb's point is also what I originally had in mind (1 is a good default because it's basically a no-op). But Colin pointed out that it would be a better UX to force the user to consider the option and enter the right value for their use case. I think he's right. It's not helpful to make it easy for users to use a scaled float that doesn't take advantage of the scaling factor (a different type would be better).

Unless anyone objects, I intend to:
* Close this PR
* Update the original issue with these notes
* Update the original issue with a note that we should also validate against a value of `1` and warn the user that this isn't very useful
* Create a blocker issue that we need to update the form hook lib to allow numeric fields to be empty instead of defaulting to 0.

Reply:
👍 sounds good. I agree that leaving it blank would be the better experience. Thanks @cjcenizal!

Reply:
I haven't checked, but my guess is that it is the EuiNumeric field that defaults to 0.

Reply:
It seems that the conversion from `""` to `0` occurs on the field `deSerializer` that we have set

```js
deserializer: (value: string | number) => +value,

// Probably needs to be

deserializer: (value: string | number) => value === '' ? '' : +value,
```